lib/HTFeed/Storage.pm

```diff
@@ -51,17 +52,18 @@ sub new {
     my $timestamp = $args{timestamp};
 
     my $self = {
-	volume         => $volume,
-	namespace      => $volume->get_namespace(),
-	objid          => $volume->get_objid(),
-	errors         => [],
-	mets_source    => $volume->get_mets_path(),
-	zip_source     => $volume->get_zip_path(),
-	zip_suffix     => '',
-	config         => $config,
-	did_encryption => 0,
-	name           => $name,
-	timestamp      => $timestamp
+        volume         => $volume,
+        namespace      => $volume->get_namespace(),
+        objid          => $volume->get_objid(),
+        errors         => [],
+        mets_source    => $volume->get_mets_path(),
+        zip_source     => $volume->get_zip_path(),
+        zip_suffix     => '',
+        config         => $config,
+        did_encryption => 0,
+        name           => $name,
```

Comment:
We should record the name of the storage as a label for all metrics recorded here.

Reply:
I've added labels to `HTFeed::JobMetrics`, so it is possible to record them. Now a matter of implementing where desired.

Reply:
Added storage name as a label value:https://github.com/hathitrust/feed/pull/128/commits/7f35bc7cfe702d0a168ac0b9be4f01f2b1a8b55e